frontend/src/components/Inquiries/InquiryForm.tsx

```diff
@@ -0,0 +1,166 @@
+import {
```

Comment:
nit:  there may be lots of forms.  you can be more speciic in the filename and component name -- eg AddOrEditInquiryForm

Reply:
Isn't this a Modal?  You should use "Modal" in the name as well

Reply:
will address